python/hsfs/feature_view.py

```diff
@@ -3437,11 +3446,218 @@ def update_from_response_json(self, json_dict: Dict[str, Any]) -> "FeatureView":
             "training_helper_columns",
             "schema",
             "serving_keys",
+            "enabled_logging",
         ]:
             self._update_attribute_if_present(self, other, key)
         self._init_feature_monitoring_engine()
         return self
 
+    def transform_batch_data(self, features):
+        self._feature_view_engine.transform_batch_data(
+            features, self.transformation_functions
+        )
+
+    def transform_feature_vector(self, features):
+        return self.transform_feature_vectors([features])
+
+    def transform_feature_vectors(self, features):
+        return self._batch_scoring_server.transform_feature_vectors(features)
+
+    def enable_logging(self) -> None:
+        """Enable feature logging for the current feature view.
+
+        This method activates logging of features.
+
+        # Example
+            ```python
+            # get feature store instance
+            fs = ...
+
+            # get feature view instance
+            feature_view = fs.get_feature_view(...)
+
+            # enable logging
+            feature_view.enable_logging()
+            ```
+
+        # Raises
+            `hsfs.client.exceptions.RestAPIError` in case the backend fails to enable feature logging.
+        """
+        return self._feature_view_engine.enable_feature_logging(self)
+
+    def log(self,
```

Comment:
There is a limitation with these arguments. It's not possible to log both untransformed and transformed features at the same time. That would be possible if we reorganize the parameters as
`
def log(self,
    transformed_features,
    untransformed_features,
    predictions,
    ...
`

Reply:
good point, may be we can change it to 
```
def log(self,
            transformed_features: Optional[Union[pd.DataFrame, list[list], np.ndarray]]=None,
            untransformed_features: Optional[Union[pd.DataFrame, list[list], np.ndarray]]=None,
            prediction: Optional[Union[pd.DataFrame, list[list], np.ndarray]]=None,
            write_options: Optional[Dict[str, Any]] = None,
            training_dataset_version: Optional[int]=None,
            hsml_model=None,
            )
```
@jimdowling agree?